hack/generate-standard-manifests.sh

```diff
@@ -121,11 +121,21 @@ VALUES_FILES=$(cd $VALUES_DIR && find * -type f -name "*.yml" )
 # Suppress potential Helm warnings about invalid permissions for Kubeconfig file
 # by throwing away related warnings.
 for values in $VALUES_FILES; do
-    $HELM template \
+    $HELM template --include-crds \
           --namespace kube-system \
           -f "$VALUES_DIR/$values" \
           $EXTRA_VALUES \
           "$ANTREA_CHART" \
           > "$OUTPUT_DIR/$values" \
           2> >(grep -v 'This is insecure' >&2)
 done
+
+# We also generate a manifest which only includes CRD resources (all of them).
+# This is useful for Antrea upgrades when using Helm.
+CRD_FILES=$(cd $ANTREA_CHART/crds && find * -type f -name "*.yaml" )
+CRD_OUTPUT_FILE="$OUTPUT_DIR/antrea-crds.yml"
```

Comment:
this file is not checked in and is not in `.gitignore`, is it intentional?

Reply:
I think it's checked in already?
Github is collapsing it, but I can see it in the diff

Reply:
Ah, I thought it was not checked in because I saw it in the diff.